docs/usage.md

```diff
@@ -109,304 +62,65 @@ They are loaded in sequence, so later profiles can overwrite earlier profiles.
 If `-profile` is not specified, the pipeline will run locally and expect all software to be installed and available on the `PATH`. This is _not_ recommended.
 
 * `docker`
-  * A generic configuration profile to be used with [Docker](http://docker.com/)
-  * Pulls software from dockerhub: [`nfcore/mag`](http://hub.docker.com/r/nfcore/mag/)
+  * A generic configuration profile to be used with [Docker](https://docker.com/)
+  * Pulls software from Docker Hub: [`nfcore/mag`](https://hub.docker.com/r/nfcore/mag/)
 * `singularity`
-  * A generic configuration profile to be used with [Singularity](http://singularity.lbl.gov/)
-  * Pulls software from DockerHub: [`nfcore/mag`](http://hub.docker.com/r/nfcore/mag/)
+  * A generic configuration profile to be used with [Singularity](https://sylabs.io/docs/)
+  * Pulls software from Docker Hub: [`nfcore/mag`](https://hub.docker.com/r/nfcore/mag/)
 * `conda`
   * Please only use Conda as a last resort i.e. when it's not possible to run the pipeline with Docker or Singularity.
   * A generic configuration profile to be used with [Conda](https://conda.io/docs/)
   * Pulls most software from [Bioconda](https://bioconda.github.io/)
-* `test`, `test_hybrid`
+* `test`, `test_hybrid`, `test_host_rm`, `test_hybrid_host_rm`
   * Profiles with a complete configuration for automated testing
   * Includes links to test data so needs no other parameters
 
-<!-- TODO nf-core: Document required command line parameters -->
-
-### `--reads`
-
-Use this to specify the location of your input FastQ files. For example:
-
```

Comment:
I do not understand what is happening. I can see that all this is moved to `nextflow_schema.json `, but how is that presented to the user? When browsing your branch I struggle to find that information.

Reply:
Hi Daniel, all the parameter information now is parsed from the `nextflow_schema.json` and displayed directly on the nf-core website, at the pipeline page. That's how this is handled. In this way we don't need to update this info in the docs every time. In the future, the help message will also be displayed directly from the doc strings in the nextflow schema.

Reply:
Sounds great, but than this information is not presented in a convenient way on the github page?
What I mean is the following: I navigate to [usage.md](https://github.com/skrakau/mag/blob/merging-template-updates/docs/usage.md) and expect there information. But there is none, even no link to the repective nf-core website. Wouldn't that ease the usage for users that do not necessarily come from the nf-core website?

Reply:
I would suggest we add the link to the nf-core website afterwards, since the corresponding section in the website does not yet exist.

Reply:
Point taken.